recipes/nnetsauce/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "nnetsauce" %}
+{% set version = "0.8.4" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/Techtonique/{{ name }}/blob/master/dist/{{ name }}-{{ version }}.tar.gz?raw=true
```

Comment:
Change to: `https://github.com/Techtonique/{{ name }}/archive/{{ version }}.tar.gz`

Reply:
@dopplershift Done. But there are still some SHA256 mismatches